### PR TITLE
Task 24

### PR DESCRIPTION
**Pull request by j-ae**
@pmvald, please review thanks.

### CODE REVIEW

File: -sass/styles/_headers.m.scss [hunk 1]
```diff
@@ -51,3 +51,54 @@
         }
     }
 }
+
+.s-profile-header {
+    height: 5*$spacer;
+    display: flex;
+    align-items: center;
+    background-color: $primary-color;
```
Comment (pmvald) [inline -sass/styles/_headers.m.scss:59]: Good

File: -sass/styles/_headers.m.scss [hunk 2]
```diff
@@ -51,3 +51,54 @@
...
+        // profile location
+        p {
+            font-size: $font-size-base;
+            color: $light-grey;
+            opacity: 0.5;
+        }
+    }
+
+    &__menu {
+        margin-left: auto;
```
Comment (pmvald) [inline -sass/styles/_headers.m.scss:89]: Good

**Review by pmvald — APPROVED** [review]

**Comment by pmvald** [comment]
@j-ae Nice and clean! Your follow-up question: why did I use h2 for the header instead of h1?

**Comment by j-ae** [comment]
@pmvald, 
**Why did I use h2 for the header instead of h1?**
h1 is for a main header and I think that you are considering the SEO(Search Engine Optimization).
Since we can change the font-size of h1,h2,h3...etc, I always thought that the header tag is for SEO or code readability.

**Pull request merged**